### ISSUE TITLE
Automatic reporting after screening or slate event

### ISSUE THREAD

**Issue by cams-goussale** [blockframes/blockframes#8684]
Could we send an automatic email to event organizer after the event with the statistics reports attached?

**Comment by RemcoSimonides** [comment]
Blocked by #8804. Let's first have the event statistics correct in the app before we start emailing them to the organizer.

Also, this issue is going to need more details. Maybe create a Sendgrid template so the dev can see there what data should be send to Sendgrid?

**Comment by Jackseed** [comment]
Not a priority anymore > closing
